ee-security/pom.xml

```diff
@@ -54,11 +54,15 @@
 
     <dependencies>
 
+        <!-- Jakarta-namespace specific deps -->
+
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>wildfly-ee</artifactId>
+            <artifactId>wildfly-ee-jakarta</artifactId>
         </dependency>
 
+        <!-- Other deps consistent with the javax.* ee-security module -->
```

Comment:
This comment is kinda out of date now ;-)

Reply:
Removed